### ISSUE TITLE
Increase contrast of links and buttons on data service

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1023]
Currently links on disaggregation report don't meet AA contrast guidelines:
![image](https://user-images.githubusercontent.com/43404358/102092852-646e0080-3e18-11eb-8130-f8a6f05b364d.png)

Links should be made darker to meet accessibility guidelines

I think it would probably make sense to have them same colour as links on Open SDG (`#1D70B8`)?

![image](https://user-images.githubusercontent.com/43404358/102093185-c3cc1080-3e18-11eb-997e-72569fea0bf7.png)

EDIT: Think this applies to whole data service actually

EDIT2: This also applies to the buttons on the data service:
![image](https://user-images.githubusercontent.com/43404358/102094712-8e282700-3e1a-11eb-846c-6d1828ccaecc.png)
